### PR TITLE
Add OpenJDK 19 & 20

### PR DESCRIPTION
**Pull request by Goooler**
https://jdk.java.net/19
https://jdk.java.net/20

### CODE REVIEW

**Comment by Goooler** [comment]
For https://github.com/compiler-explorer/infra/pull/986.

File: etc/config/kotlin.amazon.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 compilers=&kotlin
 compilerType=kotlin
 versionFlag=-version
-objdumper=/opt/compiler-explorer/jdk-18.0.0/bin/javap
+objdumper=/opt/compiler-explorer/jdk-19.0.2/bin/javap
```
Comment (Goooler) [inline etc/config/kotlin.amazon.properties:4]: Kotlin 1.8.20 hasn't support Java 20, I'm not sure if we can use JDK 20 here.

**Review by mattgodbolt — APPROVED** [review]
LGTM, thanks!

**Comment by mattgodbolt** [comment]
This is now live.

File: etc/config/kotlin.amazon.properties (hunk 1 above)
Comment (Goooler) [reply etc/config/kotlin.amazon.properties:4]: Now it's in #5280.

**Pull request merged**